_templates/component/new/wdio-spec.ejs.t

```diff
@@ -0,0 +1,16 @@
+---
+to: src/components/<%= name %>/test/wdio/default.ts
+---
+
+import { assertScreenMatches } from "../../../../utils/webdriverutils"
+import stories from "../../../../../stories.json"
+
+describe("Stories for ${<%= name %>}", () => {
+  stories['<%= name %>'].forEach(story => {
+    it(`should render the story for '<%= name %>' with ${story.name}`, () => {
+      browser.url(`/iframe.html?id=<%= name.toLowerCase() %>--${story.name}`);
+      const screenshots = browser.checkDocument()
+      assertScreenMatches(screenshots)
+    })
+  })
+})
```

Comment:
I _think_ the ideal situation would be that these specs don't live in the component dirs themselves but is dynamically generated/ran at the time of the test in a single high-level place. We've kind of all been picturing slightly different things. We should sit down and outline exactly how we envision this working 👍 

cc @blaketarter

Reply:
@JakeLaCombe and I talked about this IRL and we are going to default to dynamically iterating over stories (possibly letting the author create local WDIO tests...maybe wait on this). The responsibility will fall on the Component author to ensure that each story can be represented statically (without animation) by using boolean ENV props, e.g. `disableAnimation={process.env.NODE_ENV === "ci"}`. 👍

Reply:
And it's done! https://github.com/cremalab/crema-app-web/pull/17/commits/69a29fbc83df8f16211cdcf0dbc780d810b7e182

Reply:
I see you have a different prop type request. I can change to that.